apps/frontend/src/components/form/FormInput.tsx

```diff
@@ -25,9 +27,28 @@ const FormInput: React.FC<FormInputProps> = ({
   id,
   description,
   selectList,
+  onChange,
+  isDisabled,
 }) => {
+
+  const handleChange = (value: any) => {
+    field.onChange({
+      target: {
+        value,
+        name: field.name,
+      },
+    });
+
+    if (onChange) {
+      onChange(value);
+    }
+  };
+
   if (variant === 'text' || variant === 'email' || variant === 'date') {
-    return <Input {...field} id={id} type={variant} />;
+    return <Input {...field} id={id} type={variant} 
+    onChange={(e) => handleChange(e.target.value)}
```

Comment:
Is it necessary to pass the onChange method to anything other than the select component?